demos/text_detection_demo/cpp/README.md

```diff
@@ -36,11 +38,20 @@ This file can be used as a parameter for [Model Downloader](../../../tools/downl
 * decoder_type = ctc
   * text-recognition-0012
   * text-recognition-0013
+  * text-recognition-0014
 * decoder_type = simple
+  * text-recognition-0015
   * text-recognition-resnet-fc
 
 > **NOTE**: Refer to the tables [Intel's Pre-Trained Models Device Support](../../../models/intel/device_support.md) and [Public Pre-Trained Models Device Support](../../../models/public/device_support.md) for the details on models inference support at different devices.
 
+> **NOTE**: In case of composite model encoder and decoder are searched automatically: this means that model encoder should have `encoder` part in its name and model decoder should have `decoder` part. In this case to run the demo specify path to the encoder model (`-m_tr` parameter) and decoder model will be searched in the same path but `encoder` would be replaced with `decoder`. E.g.:
+> ```
+> model-text-recognition-0015:
+>   model_encoder.xml
+>   model_decoder.xml
```

Comment:
Different subfolders is not a problem by itself. Automatic decoder search just replaces all `encoder` substring entries to `decoder`. If downloader loads model in text-rec-0015/encoder/encoder_model.xml & text-rec-0015/decoder/decoder_model.xml it is ok. What I wanted to say, if encoder is in `text-rec-0015/encoder/encoder_model.xml` and decoder in `text-rec-0015/encoder/model.xml` (for example) this would not work.